rules/0.17-allow-new-players.py

```diff
@@ -13,17 +14,24 @@ def should_allow(pr):
   
   (points_user, points_name, points_change) = bonuses[0]
   
-  if points_user in util.users():
-    raise Exception('Cannot create an existing user')
-  
   if points_name != 'initial':
-    raise Exception('New player bonus value must be called "initial"')
-
+    raise Exception('New player bonus value is called %s instead of "initial"' % points_name)
+    
   if points_change < 0:
     raise Exception('Points cannot be negative')
 
   if points_change > max_start_bonus:
     raise Exception('%s initial points exceeds maximum starting value of %s points' %
                     (points_change, max_start_bonus))
+  
+  bonus_directory = os.path.join('players', points_user, 'bonuses')
+  if os.path.isdir(bonus_directory):
+    for bonus in os.listdir(bonus_directory):
+      if bonus != 'initial':
+        raise Exception('%s already has bonuses' % points_user)
+    
+  response = util.request('https://www.jefftk.com/nomic-github/users/%s' % points_user)
+  if response.status_code != 200:
+    raise Exception('%s is not a real GitHub user' % points_user)
```

Comment:
better idea: require the PR to be authored by the user who wants to be added.

Reply:
Makes sense.